tests/beta/test_default_tensor_tf.py

```diff
@@ -97,74 +102,77 @@ def _init_state(n):
 #####################################################
 
 
+@pytest.mark.parametrize("rep", ("exact", "mps"))
 class TestApply:
     """Test application of PennyLane operations."""
 
-    def test_basis_state(self, tol):
+    def test_basis_state(self, tol, rep):
         """Test basis state initialization"""
-        dev = DefaultTensorTF(wires=4)
+        dev = DefaultTensorTF(wires=4, representation=rep)
         state = np.array([0, 0, 1, 0])
 
         dev.execute([qml.BasisState(state, wires=[0, 1, 2, 3])], [], {})
 
-        res = dev._state.numpy().flatten()
+        res = dev._state().numpy().flatten()
         expected = np.zeros([2 ** 4])
         expected[np.ravel_multi_index(state, [2] * 4)] = 1
 
         assert np.allclose(res, expected, atol=tol, rtol=0)
 
-    def test_qubit_state_vector(self, init_state, tol):
+    def test_qubit_state_vector(self, init_state, tol, rep):
         """Test qubit state vector application"""
-        dev = DefaultTensorTF(wires=1)
+        dev = DefaultTensorTF(wires=1, representation=rep)
         state = init_state(1)
 
         dev.execute([qml.QubitStateVector(state, wires=[0])], [], {})
 
-        res = dev._state.numpy().flatten()
+        res = dev._state().numpy().flatten()
         expected = state
         assert np.allclose(res, expected, atol=tol, rtol=0)
 
-    def test_invalid_qubit_state_vector(self):
+    def test_invalid_qubit_state_vector(self, rep):
         """Test that an exception is raised if the state
         vector is the wrong size"""
-        dev = DefaultTensorTF(wires=2)
+        dev = DefaultTensorTF(wires=2, representation=rep)
         state = np.array([0, 123.432])
 
-        with pytest.raises(ValueError, match=r"State vector must be of length 2\*\*wires"):
+        with pytest.raises(
+            ValueError, match=r"can apply QubitStateVector only to all of the 2 wires"
```

Comment:
Just noticing this now, but this sounds a bit weird. "can _only_ apply QubitStateVector to all 2 wires" sounds better

Reply:
I'll have to disagree here, I think it's fine as is